src/client/cli/cmd/exec.cpp

```diff
@@ -24,6 +24,26 @@
 namespace mp = multipass;
 namespace cmd = multipass::cmd;
 
+namespace
+{
+const QString work_dir_option_name{"working-directory"};
+const QString no_dir_mapping_option{"no-map-working-directory"};
+
+bool is_dir_mounted(const QStringList& split_current_dir, const QStringList& split_source_dir)
```

Comment:
Could make this `auto` since it's in the anonymous namespace and it's clear `bool`'s are being returned.

Reply:
Done.